src/controller/tests/TestServerCommandDispatch.cpp

```diff
@@ -118,7 +121,8 @@ CHIP_ERROR TestClusterCommandHandler::EnumerateAcceptedCommands(const ConcreteCl
     }
 
     // We just have one command id.
-    callback(Clusters::UnitTesting::Commands::TestSimpleArgumentRequest::Id, context);
+    ReturnErrorOnFailure(builder.EnsureAppendCapacity(1));
```

Comment:
this could `referenceExisting` since there is a fixed command list and not use RAM.